app/services/AuditService.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+
+import java.time.Instant
+import java.util.UUID
+
+import audit.events.{AuditEventConstants, ContactDetailsAuditEventDetail, EmailMismatchEventDetail, EmailVerifiedEventDetail, QuestionnaireAuditEvent}
+import javax.inject.{Inject, Singleton}
+import models.{CompanyContactDetails, QuestionnaireModel}
+import play.api.libs.json.{Json, Writes}
+import play.api.mvc.{AnyContent, Request}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.audit.AuditExtensions.auditHeaderCarrier
+import uk.gov.hmrc.play.audit.http.connector.{AuditConnector, AuditResult}
+import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class AuditServiceImpl @Inject()(val auditConnector: AuditConnector) extends AuditService {
+}
```

Comment:
Remove the open and close parenthesise as this looks like it causes a ScalaStyle error which is why the Jenkins PR builder fails:
```suggestion
class AuditServiceImpl @Inject()(val auditConnector: AuditConnector) extends AuditService
```